### PR TITLE
Basic table Prealgebra 2e

### PR DESCRIPTION
**Pull request by ajluk**
Issue:  #2319

### CODE REVIEW

File: styles/designs/corn/parts/_table-shapes.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
+@import '_table-components';
+@import '_caption-components';
+
+@include create_shape('carnival:::BasicTableShape',  (
```
Comment (wilkus27) [inline styles/designs/corn/parts/_table-shapes.scss:4]: It should be `corn` here, not `carnival`.

File: styles/books/dev-math/book.scss [hunk 2]
```diff
@@ -13,3 +13,6 @@ $ChapterIntroType: fullWidth;
 @import '../../templates/template2/settings.scss';
 
 @include use('BookRoot', "common:::BookRoot");
+
+//Tables
+@include use('BasicTable', 'carnival:::BasicTableShape');
```
Comment (wilkus27) [inline styles/books/dev-math/book.scss:18]: It should be corn here, not carnival.

File: styles/templates/template2/parts/_table-settings.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
+@import 'styles/designs/corn/parts/table-shapes';
+@import 'templates/template2/colors';
+@import 'templates/template2/typography';
```
Comment (wilkus27) [inline styles/templates/template2/parts/_table-settings.scss:3]: I think we don't need those imports here. `table-shapes` file is imported in `design.scss`, `colors` and `typography` in `settings.scss` file.

Comment (ajluk) [reply styles/templates/template2/parts/_table-settings.scss]: Thanks Natalia I have removed them.

File: styles/books/dev-math/book.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
+$PagesWithBands: (
+  (pageName: eoc, firstSelector: '.os-eoc.os-glossary-container', generalSelector: '.os-eoc'),
+  (pageName: eob, firstSelector: '.os-eob[data-type="composite-chapter"]', generalSelector: '.os-eob'),
+  (pageName: appendix, firstSelector: '.appendix', generalSelector: '.appendix'),
+);
+
+$bandColor: #1E7E91;
```
Comment (Stackblocks) [inline styles/books/dev-math/book.scss:7]: Should $bandColor be set in colors.scss?

File: styles/designs/corn/parts/_table-components.scss [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+        padding-right: table-h-spacing(1),
+        padding-left: table-h-spacing(1),
+        color: enum('ValueSet:::REQUIRED'),
+        vertical-align: top,
+    )
+);
+
+$Table__Head__Row: empty_wrapper(TableHead, ' > tr');
+
+$Table__Row__Data--Last: (
```
Comment (Stackblocks) [inline styles/designs/corn/parts/_table-components.scss:79]: Should this be `$Table__Row--Last__Data`?

File: styles/books/dev-math/book.scss (hunk 4 above)
Comment (wilkus27) [reply styles/books/dev-math/book.scss:7]: We can't do this. All variables used in `bands.scss` file must be defined in `book.scss` file before those imports:

image

That's why we can't use colors defined in `colors.scss` file.

**Pull request merged**